### ISSUE TITLE
Support for basic hair

### ISSUE THREAD

**Issue by blented** [ChaosGroup/vray-for-houdini#138]
Basic hair doesn't seem to render correctly.  Is there a special flag that needs to be set perhaps?  The same hair renders correctly when cached out as an alembic and read back in as a VRayProxy.

Test scene:
https://drive.google.com/open?id=0B-R03L35FWUSREI0eGZRVlNDaVU

Hair in Houdini (renders as closed polygons):
https://drive.google.com/open?id=0B-R03L35FWUST21KOEhTRVVEajg

Hair from Alembic via VRayProxy (what I would consider correct):
https://drive.google.com/open?id=0B-R03L35FWUSMUtVUjlsM2h1aE0

There is a VRay GeomMayaHair node that generates this output when plugged in to the tree (also not correct but nice that it's subdivision smoothed):
https://drive.google.com/open?id=0B-R03L35FWUSSDZpdm45Z08xU0E

Houdini 15.459
VRay Nightly from 11/14
Windows 10

**Comment by bdancer** [comment]
This should fix width for GeomMayaHair 3aadaf9fc89043f193b5587f33af98274a810d5c.
Actually, it should work without GeomMayaHair, I'll check what goes wrong.

**Comment by bdancer** [comment]
Btw, seems like alembic rop also looks for "width" attr, if I leave only "pscale" I got very strange results.

**Comment by bdancer** [comment]
83e9ffb3851fd260ca3f3bb7bd460b0111ff1d06 will make it possible to export hair with GeomMayaHair node, but we'll refactor this part in a better way.

**Comment by millag** [comment]
This is now fixed. We've removed GeomMayaHair SOP node and the exporter will now interpret directly the output geometry. All curves and open poly geometry is passed to V-Ray as hair. Parameters for subdivision, width, etc. that were previously found on the SOP can now be added through Edit Parameters Interface->Rendering Parameters->V-Ray->V-Ray Hair as optional rendering parameters to the parent OBJ node.
